tests/TestCase/Controller/Component/SecurityComponentTest.php

```diff
@@ -789,32 +788,6 @@ public function testValidatePostHidden(): void
         $this->assertNull($result);
     }
 
-    /**
-     * testValidatePostWithDisabledFields method
-     *
-     * @return void
-     * @triggers Controller.startup $this->Controller
-     */
-    public function testValidatePostWithDisabledFields(): void
```

Comment:
Do we no longer support disabled fields?

Reply:
Disabled fields were deprecated and super seeded by unlocked fields in 3.0.0 itself. It's no longer documented even. Here's the related commit from 2011 https://github.com/cakephp/cakephp/commit/c1363499191eef0a1829295ba1afb9e5f12807c7.